OWNERS_ALIASES

```diff
@@ -173,10 +173,11 @@ aliases:
     - carolynvs
     - bradamant3
   features-maintainers:
-    - idvoretskyi
-    - justaugustus
-    - kacole2
+    - idvoretskyi # 1.11 Features Lead
+    - justaugustus # 1.12 Features Lead
+    - kacole2 # 1.13 Enhancements Lead
+    - claurence # 1.14 Enhancements Lead
   pm-maintainers:
```

Comment:
@justaugustus can you please add me here as well (as a SIG-PM chair)? 

PS. I'm fine with removing my name from above as a 1.11 features lead.